src/org/usfirst/frc4904/robot/commands/Climb.java

```diff
@@ -23,7 +23,7 @@ protected void execute() {
 		set(Math.max(0, axis.get())); // No, really, always feed positive values
 										// to the climber motor
 		if (axis.get() >= Climb.MINIMUM_AXIS_VALUE_TO_FLIP_GEARIO_RAMP && RobotMap.Component.gearIO.isNotOverridden()) {
-			RobotMap.Component.gearIO.setRampState(GearIO.RampState.RETRACTED);
+			new SetRampState(GearIO.RampState.RETRACTED).start();
```

Comment:
In a perfect world, this would probably be on a separate pull request, but I do not think that it is too much of a problem.